modules/client/src/client-common/interfaces/common.ts

```diff
@@ -55,3 +48,16 @@ export interface IInterfaceParams {
   functionName: string;
   hash: string;
 }
+
+export interface IEncodingResult {
+  to: string;
+  value: BigNumberish;
+  data: Uint8Array;
+}
+
+export interface IDecodingTX {
```

Comment:
These look more like a type than like an interface to me. If there were methods that could match different implementations under the same umbrella, that would feel good here... but being a precise object I would use a type.

Reply:
done